cluster/saltbase/salt/kubelet/default

```diff
@@ -15,6 +15,15 @@
   {% set api_servers = "--api_servers=https://" + ips[0][0] + ":6443" -%}
 {% endif -%}
 
+# Disable registration for the kubelet running on the master on GCE.
```

Comment:
Eventually we do want to register the master kubelet. Registration just needs to be retried if the apiserver doesn't respond. And, for now, we want scheduling to be disabled for the master node. #6428
cc @dchen1107 @vishh